### DISCUSSION TITLE
Windows PowerShell Error

### DISCUSSION THREAD

**Discussion by OOF-King** [KadeArchive/Kade-Engine#2895] [Q&A]
I keep getting this error when inputting 'lime test windows' : 
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/fs/PolymodFileSystem.hx:19: characters 16-18 : Expected )
Anybody know a way to fix this?
![Capture71](https://user-images.githubusercontent.com/61514339/161382116-e2757d2e-8200-4d18-b90a-4520f54c4e77.PNG)

**Comment by BoloVEVO** [comment]
Write `haxelib remove polymod `and then `haxelib git polymod https://github.com/MasterEric/polymod.git`. That should fix your problem.

**Comment by ActualeAnEggDog** [comment]
change your haxe to latest version (my version is 4.1.5)
